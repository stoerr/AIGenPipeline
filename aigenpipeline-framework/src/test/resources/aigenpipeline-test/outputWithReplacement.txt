this should be left alone at the beginning.
yes yes
<!-- start of thereplacedpart -->
this should be overwritten
<!-- end of thereplacedpart AIGenVersion() -->
this should be left alone at the end.
yes
