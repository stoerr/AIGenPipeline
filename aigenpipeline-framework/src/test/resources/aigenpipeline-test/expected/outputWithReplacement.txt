this should be left alone at the beginning.
yes yes
<!-- start of thereplacedpart -->
// AIGenVersion(28bb7de4, prompt.txt-45366fcf, input.txt-7e10dd2a, inputWithVersion.txt-1.1)

Response to:
{
  "model": "gpt-4o",
  "messages": [
    {
      "role": "system",
      "content": "You are an expert programming assistant who follows the users instructions exactly and to the letter.\nYou observe the rules of clean code, KISS, YAGNI, and DRY and love good documentation and well documented code.\nDo never ever give any introductory or concluding text, just the requested output, except if explicitly asked for.\nDo not quote the output, just print the raw output.\n\nIMPORTANT: If something important in the instructions is unclear, there is information missing,\nyou have any important questions or after printing the response you notice that something was wrong, then append to the file a paragraph starting with\n`EMCIF(GenAIPipeline) ` and a description of that problem. Do not state mere suggestions for improvement\nof the output, tough.\n"
    },
    {
      "role": "user",
      "content": "Retrieve the content of the input file 'src/test/resources/aigenpipeline-test/input.txt'"
    },
    {
      "role": "assistant",
      "content": "This is a test input file for AIGenerationTask."
    },
    {
      "role": "user",
      "content": "Retrieve the content of the input file 'src/test/resources/aigenpipeline-test/inputWithVersion.txt'"
    },
    {
      "role": "assistant",
      "content": "--\nversion: \n--\nThis is a test input file for AIGenerationTask.\n"
    },
    {
      "role": "user",
      "content": "This is a test prompt for AIGenerationTask."
    }
  ],
  "temperature": 0.0,
  "max_tokens": 2048
}
<!-- end of thereplacedpart AIGenVersion() -->
this should be left alone at the end.
yes
